user_docs/en/userGuide.t2t

```diff
@@ -546,6 +546,27 @@ A gesture allows cycling through the various speech modes:
 
 If you only need to switch between a limited subset of speech modes, see [Modes available in the Cycle speech mode command #SpeechModesDisabling] for a way to disable unwanted modes.
 
+++ Sound split modes ++[SpeechModes]

```

Comment:
Then perhaps I should remove the combobox from settings as well - if speech mode doesn't have a combo box then why sound split needs one?

Reply:
> Then perhaps I should remove the combobox from settings as well - if speech mode doesn't have a combo box then why sound split needs one?

Usually, NVDA settings are reachable either through the GUI (settings dialog) or through gestures (assigned or unassigned). The features that are not modifiable through settings dialogs are the exception.

Having the feature controllable in the settings dialog allows:
* saving the config on exit and restoring it at NVDA startup
* using different profiles
* a better discoverability of the feature since its control is visible in the GUI; someone can use it even without knowing the shortcut NVDA+alt+S

IMO, this is suitable for sound split:
* maybe this is not very common but we cannot exclude that someone wants to have sound split always enabled, so also at NVDA startup
* profiles may be useful in the following cases:
  * VLC profile using: NVDA left and application in both channels
  * Teams / Skype / any conferencing application using : NVDA left and application right

Reply:
I think as cyrille outlined here, the paragraph is not necessary. The documentation in settings should be sufficient